test/fixtures/books.yml

```
@@ -1,9 +1,6 @@
 # Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
-one:
-  title: MyString
-  memo: MyText
-
-two:
-  title: MyString
-  memo: MyText
+harry_potter_book:
```

```suggestion
harry_potter:
```

`books(:harry_potter_book)` みたいに書くので _book は冗長かもしれませんね

---

修正いたしました!